### ISSUE TITLE
[Feature] Support Jenkins plugin in KubeVela

### ISSUE THREAD

**Issue by FogDong** [oam-dev/kubevela#3010]
**Is your feature request related to a problem? Please describe.**

KubeVela can be integrated with other CI tools like Jenkins or Argo, we can have Jenkins as our plugin to support CI.
If anyone want to help, please contact me for more info.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by du86796922** [comment]
support，In fact, there has been a similar demand recently

**Comment by tangwz** [comment]
I'm a newbee, it's good first issue.May I have a try?

**Comment by wonderflow** [comment]
@tangwz sure, welcome

**Comment by FogDong** [comment]
> I'm a newbee, it's a good first issue.May I have a try?

Sure! If you have any questions, please contact me: wuwuglu19@gmail.com or WeChat:wuwugludtx

**Comment by FogDong** [comment]
Is there any progress or problem? @tangwz

**Comment by tangwz** [comment]
Should I add this plugin in https://github.com/oam-dev/catalog? @FogDong

**Comment by wonderflow** [comment]
@tangwz Hi, any progress on this issue?

**Comment by FogDong** [comment]
> Should I add this plugin in https://github.com/oam-dev/catalog? @FogDong

Yes, you can refer to this doc: https://kubevela.io/docs/next/platform-engineers/addon/intro

**Comment by Sajiyah-Salat** [comment]
is this issue still valid?

**Comment by FogDong** [comment]
@Sajiyah-Salat Yes it is still valid, are you interested in this issue?

**Comment by Sajiyah-Salat** [comment]
Yes I am. But I don't have any knowledge about kubevela project. I am a contributor in K8s. And have good knowledge of open source. Kindly provide me resources to understand this repo.

**Comment by FogDong** [comment]
@Sajiyah-Salat You can checkout the [official doc](https://kubevela.io/docs/) to learn more about KubeVela, and [addon doc](https://kubevela.io/docs/platform-engineers/addon/intro) to know how to customize an addon.

**Comment by satyazzz123** [comment]
> **Is your feature request related to a problem? Please describe.**
> 
> KubeVela can be integrated with other CI tools like Jenkins or Argo, we can have Jenkins as our plugin to support CI. If anyone want to help, please contact me for more info.
> 
> **Describe the solution you'd like**
> 
> **Describe alternatives you've considered**
> 
> **Additional context**


@FogDong I want to work on this issue. In this issue we basically have to add jenkins and other such CI tools plugins? Such plugins don't exist in kubeVela? Did I get it right?

**Comment by FogDong** [comment]
Yes exactly, thanks for the help in advance. @satyazzz123

**Comment by priyanshsao** [comment]
i would like to work on this issue. @satyazzz123 are you still working on this issue?
